### ISSUE TITLE
Missing momentum in documentation of ResourceApplyCenteredRMSProp

### ISSUE THREAD

**Issue by pooyadavoodi** [tensorflow/tensorflow#42502]
The issue is for this page: https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/resource-apply-centered-r-m-s-prop

`momentum` is missing from the list of arguments.

Current doc has this:

```
scope: A Scope object
var: Should be from a Variable().
mg: Should be from a Variable().
ms: Should be from a Variable().
mom: Should be from a Variable().
lr: Scaling factor. Must be a scalar.
rho: Decay rate. Must be a scalar.
epsilon: Ridge term. Must be a scalar.
grad: The gradient.
```

This should be instead like this (momentum added after rho):

```
scope: A Scope object
var: Should be from a Variable().
mg: Should be from a Variable().
ms: Should be from a Variable().
mom: Should be from a Variable().
lr: Scaling factor. Must be a scalar.
rho: Decay rate. Must be a scalar.
momentum: momentum scale. Must be a scalar.
epsilon: Ridge term. Must be a scalar.
grad: The gradient.
```

### Clear description
This could be due to a bug in the scripts that auto-generate the api-def.

This is the op registration from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/training_ops.cc#L1056-L1069 

```
REGISTER_OP("ResourceApplyCenteredRMSProp")
    .Input("var: resource")
    .Input("mg: resource")
    .Input("ms: resource")
    .Input("mom: resource")
    .Input("lr: T")
    .Input("rho: T")
    .Input("momentum: T")
    .Input("epsilon: T")
    .Input("grad: T")
    .Attr("T: numbertype")
    .Attr("use_locking: bool = false")
    .SetShapeFn(
        ApplyCenteredRMSPropShapeFn</*is_sparse=*/false, /*is_resource=*/true>);
```

**Comment by ashwin-phadke** [comment]
I created a change for [ApplyCenteredRMSProp](https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/apply-centered-r-m-s-prop) whereas @pooyadavoodi has mentioned that momentum is missing in ResourceApplyCenteredRMSProp documentation. 

It slipped my sight while making the change.

1) ApplyCenteredRMSProp - needs to be changed too?  - @pooyadavoodi 
2) I will create the change for ResourceApplyCenteredRMSProp.

**Comment by lamberta** [comment]
Thank you @ashwin-phadke !

@MarkDaoust Is it possible to build the Markdown for the OSS C++ API docs?

**Comment by MarkDaoust** [comment]
Rebuild them to publish this change? 

No. This just went in, and the cc docs only show latest stable (no nightly option). This will get published to the site with 2.4.

**Comment by pooyadavoodi** [comment]
> 1. ApplyCenteredRMSProp - needs to be changed too?  - @pooyadavoodi
I think so. I thought the pbtxt files are auto-generated but @mihaimaruseac says no. It would be good to automate this to avoid future bugs like this.

**Comment by mihaimaruseac** [comment]
This should be fixed by #42624
